tinylog-impl/src/test/java/org/tinylog/util/UdpSyslogServer.java

```diff
@@ -0,0 +1,79 @@
+/*
+ * Copyright 2023 Piotr Karlowicz
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not use this file except in compliance with
+ * the License. You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software distributed under the License is distributed on
+ * an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the License for the
+ * specific language governing permissions and limitations under the License.
+ */
+
+package org.tinylog.util;
+
+import java.io.IOException;
+import java.net.DatagramPacket;
+import java.net.DatagramSocket;
+import java.net.SocketException;
+import java.nio.charset.Charset;
+
+import org.tinylog.Level;
+import org.tinylog.provider.InternalLogger;
+
+public class UdpSyslogServer extends Thread {
```

Comment:
Please start and end the body of the class with an empty line, too.

Reply:
Updated